components/singleEvent.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+export default () => {
+
+    const [event, setEvent] = useState([]);
+
+    const { uuid } = useParams()
+
+    const fetchData = async () => {
+        const result = await fetch(
+            `/api/events/${uuid}`
+        ).then((res) => res.json());
+
+        setEvent(result.event)
+
+    }
+
+    useEffect(() => {
+        fetchData()
+    }, []);
+
+    return (
+        <div>
+            <h5>Details for  <b>"{event.name}"</b> scheduled for <b>{event.date}</b></h5>
+            <p><b>Start time: </b> {event.start_time}</p>
```

Comment:
Non-blocking feedback but I think we can replace all of the `b` tags with `strong`, it feels more appropriate given the context. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong

Reply:
Interesting! Good to know. Thank you for this. I'm thinking of just making changes via styling with CSS, actually, so those tags will probably be removed. Do you tend to make CSS changes at the very end? I just wanted to get the basic functionality before making it look nice.

Reply:
Yea, I'll typically get the functionality down and then make it look cute.

Reply:
Got it! Hahaha. I feel like making everything look cute can be its own Jira ticket.